rfcs/RFC Simulation Interfaces.md

```diff
@@ -701,27 +363,26 @@ Note that the ROS 2 community probably propose suitable replacement messages in
 
 - This should also provide detailed information on compatibility with different hardware platforms. -->
 
-
-### What are the advantages of the feature?
+## What are the advantages of the feature?
 <!-- - Explain the advantages for someone to use this feature -->
- - Leading standardization of simulation in the robotics domain,
- - easier usage of O3DE in robotics and ML pipelines,
- - improve the testability of the ROS 2 features,
- - adjustments to EOL of `gazebo_msgs`
+- Adoption of the simulation standard in the robotics domain in O3DE.
+- Improving the user experience of robotics and ML pipelines in O3DE.
+- Improvement of the ROS 2 features' testability.
+- Adjustments to the end-of-life of `gazebo_msgs`.
 
-### What are the disadvantages of the feature?
+## What are the disadvantages of the feature?
 <!-- - Explain any disadvantages for someone to use this feature -->
-Significant increase in code base size for ROS2 Gem.
+Significant increase in the code base size for ROS 2 Gem.
 
 ### How will this be implemented or integrated into the O3DE environment?
 <!-- - Explain how this will be integrated within codebase of O3DE and any special library or technical stack requirements. -->
-It was explained in great detail in the section [ROS 2 API](#ros-2-api).
+The *Simulation Interfaces* feature will build over the ROS 2 Gem. In particular, three separate tools for managing the *spawnables*, the simulation and the named poses will be implemented as O3DE *System Components*.  The tools will be callable using O3DE bus system and via ROS 2 middleware. The latter will be decoupled from the *Simulation Interfaces* implementation to ensure the system can be used with different frameworks in the future. More details about the design are given in section [ROS 2 API](#ros-2-api).
 
 
 ### Are there any alternatives to this feature?
 <!-- - Provide any other designs that have been considered. Explain what the impact might be to not doing this.
 - If there is any prior art or approaches with other frameworks in the same domain, explain how they may have solved this problem or implemented this feature. -->
-The effort needs to be taken to adjust the existing code to the deprecation of Gazebo Classic and `gazebo_msgs`.
+The effort needs to be taken to adjust the existing code to the deprecation of Gazebo Classic and `gazebo_msgs`. *ROS 2 Simulation Interfaces* are the upcoming simulation standard that was primarily designed by O3DE *sig-simulation* developers. It will be integrated into all simulation engines and, therefore, it makes no sense to 
```

Comment:
```suggestion
The effort needs to be taken to adjust the existing code to the deprecation of Gazebo Classic and `gazebo_msgs`. *ROS 2 Simulation Interfaces* are the upcoming simulation standard that was primarily designed by O3DE *sig-simulation* developers. It will be integrated into all simulation engines and, therefore, it makes no sense to omit introduction. 
```